Jenkinsfile

```diff
@@ -267,33 +270,47 @@ pipeline {
             node('master') {
                 script {
                     deleteDir()
-                    def logServer = [
-                        addr: env.LOG_SERVER,
-                        user: env.LOG_SERVER_USER,
-                        folder: env.LOG_SERVER_FOLDER,
-                        rootDir: env.LOG_ROOT_DIR
-                    ]
-                    def destDir = decideLogsDestination(logServer, env.ZUUL_UUID)
+                    def relLogsDstDir = logsRelPathBasedOnTriggerSource(env.JOB_NAME,
+                        env.BUILD_NUMBER, env.ZUUL_UUID)
 
                     dir('to_publish') {
                         unstash 'processedTestReports'
                         def logFilename = 'log.txt.gz'
-                        obtainLogFile(env.JOB_NAME, env.BUILD_ID, logFilename)
-                        publishToLogServer(logServer, ".", destDir)
+                        createCompressedLogFile(env.JOB_NAME, env.BUILD_NUMBER, logFilename)
+
+                        def auth = sshAuthority(env.LOG_SERVER_USER, env.LOG_SERVER)
+                        def src = "."
+                        def dst = logsDirInFilesystem(env.LOG_ROOT_DIR, env.LOG_SERVER_FOLDER, relLogsDstDir)
+                        publishToLogServer(auth, src, dst)
                     }
 
-                    def logsURL = getLogsURL(logServer, env.ZUUL_UUID)
 
+                    def fullLogsURL = logsURL(env.LOG_SERVER, env.LOG_SERVER_FOLDER, relLogsDstDir)
+                    def logDestMsg = "Full logs URL: ${fullLogsURL}"
+                    echo(logDestMsg)
```

Comment:
This `echo` can be removed.

Reply:
I think we wanted to print this so that we can watch Jenkins console log and go to log server URL.